oap-server/server-bootstrap/src/main/resources/oal/java-agent.oal

```diff
@@ -25,4 +25,7 @@ instance_jvm_memory_noheap_max = from(ServiceInstanceJVMMemory.max).filter(heapS
 instance_jvm_young_gc_time = from(ServiceInstanceJVMGC.time).filter(phrase == GCPhrase.NEW).sum();
 instance_jvm_old_gc_time = from(ServiceInstanceJVMGC.time).filter(phrase == GCPhrase.OLD).sum();
 instance_jvm_young_gc_count = from(ServiceInstanceJVMGC.count).filter(phrase == GCPhrase.NEW).sum();
-instance_jvm_old_gc_count = from(ServiceInstanceJVMGC.count).filter(phrase == GCPhrase.OLD).sum();
\ No newline at end of file
+instance_jvm_old_gc_count = from(ServiceInstanceJVMGC.count).filter(phrase == GCPhrase.OLD).sum();
+instance_jvm_thread_live_count = from(ServiceInstanceThreadGC.liveCount).sum();
+instance_jvm_thread_daemon_count = from(ServiceInstanceThreadGC.daemonCount).sum();
+instance_jvm_thread_peak_count = from(ServiceInstanceThreadGC.peakCount).sum();
```

Comment:
Why do you `sum` the count? This value could be collected multiple times, then the thread number(sum value) would be very large.

Reply:
Sorry, I was wrong here